test/testsupport/stubs/BarsStub.scala

```diff
@@ -30,7 +30,7 @@ object BarsStub {
 
     def accountNumberNotWellFormatted(): StubMapping = stubOk(validateUrl, ValidateJson.accountNumberNotWellFormatted)
 
-    def sortCodeNotPresentOnEiscd(): StubMapping = stubOk(validateUrl, ValidateJson.sortCodeNotPresentOnEiscd)
+    def sortCodeNotPresentOnEiscd(): StubMapping = stubOk(validateUrl, ValidateJson.accountNumberNotWellFormatted)
```

Comment:
Was this a mistake?

Reply:
oh yeah, whoops I'll revert it back

Reply:
not sure how that happened - corrected now [here](https://github.com/hmrc/essttp-frontend/pull/107/commits/175f2805bac302d37abbab5bb5c0f2684eb2904c)